combine/combine.go

```diff
@@ -24,13 +24,31 @@ import (
 // To do so, the user must prepare inputDir as follows:
 //   - place the ".charon" directory in inputDir, renamed to another name
 //
-// Combine will create a new directory named after the public key of each validator key reconstructed, containing each
-// keystore under the "validator_keys" subdirectory.
-func Combine(ctx context.Context, inputDir string, force bool) error {
+// Combine will create a new directory named after "outputDir", which will contain Keystore files.
+func Combine(ctx context.Context, inputDir, outputDir string, force bool) error {
 	log.Info(ctx, "Recombining key shares",
 		z.Str("input_dir", inputDir),
+		z.Str("output_dir", outputDir),
 	)
```

Comment:
suggest logging this after changing to absolute paths.

Reply:
good catch!